accord-core/src/test/java/accord/burn/random/IntRange.java

```diff
@@ -0,0 +1,21 @@
+package accord.burn.random;
+
+import java.util.Random;
+
+public class IntRange implements RandomInt
+{
+    private final int min, upperBound;
```

Comment:
upperBound is confusing. Perhaps `maxDelta`, or `range`?

Reply:
I prefer `maxDelta` so went with that